docs/Glossary.md

```diff
@@ -0,0 +1,35 @@
+# Glossary
+
+A glossary for(or used in) ONE-vscode
+
+## Concepts of ONE-vscode that You Must Know
+
+There are concepts only used in ONE-vscode. Let's investigate them.
+
+### Backend
+
+The word `Backend` is used to a concept to wrap `Toolchain` and others that related to `Toolchain`.
+
+### Toolchain
+
+`Toolchain` is a collection of tools. Now you could think it as `ONE` itself.
+
+### ONE Explorer View
+
+`ONE Explorer View` is a tree view for NN models and config file.
+
+### Compile
+
+`Compile` in ONE-vscode is doing `Compile Steps` in a config file with `Toolchain`.
+
+#### Compile Steps
+
+- Import: Import a NN model to convert it to `circle` model.
+- Optimization: Try optimizing as possible by using `circle2circle`.
+- Quantization: Quantize `circle` model.
+- Codegen: Generate code as output for backend.
+- Profile: Profile it.
```

Comment:
```suggestion
- Profile: Profile output with backend.
```

IMHO it's better to avoid use `it`. How about replacing `it` with specific objective word? [Google Technical Writing](https://developers.google.com/tech-writing/one/words#it_and_they) says `it` can confuse the reader. 
There must be much better way to describe profile step, but this is my best... If you have another way to write down, it will be better than `Profile it`.